src/core/server/rendering/rendering_service.tsx

```diff
@@ -114,7 +121,13 @@ export class RenderingService {
               uiSettings: settings,
             },
             branding: {
-              logoUrl: validLogoUrl,
+              logoUrl:
+                isLogoUrlValid === true ? opensearchDashboardsConfig.branding.logoUrl : undefined,
```

Comment:
`isLogoUrlValid === true` required? could it just be `isLogoUrlValid ? opensearchDashboardsConfig.branding.logoUrl : undefined`

Reply:
was doing this but vscode keeps giving me error. The error went away after I restart vscode.. will delete ===true